src/theme/components/notification.ts

```diff
@@ -2,31 +2,34 @@ import { NotificationTypes } from "../../components/Notification/NotificationTyp
 
 const Notification = {
   parts: ["container", "dismissibleButton", "icon"],
-  baseStyle: ({ centered, noMargin, notificationType }) => {
+  baseStyle: ({ centered, dismissible, noMargin, notificationType }) => {
     let bg = "ui.status.primary";
-    if (notificationType === NotificationTypes.Announcement) {
-      bg = "ui.gray.light-cool";
-    } else if (notificationType === NotificationTypes.Warning) {
-      bg = "ui.gray.xx-light-cool";
+    if (
+      notificationType === NotificationTypes.Announcement ||
+      notificationType === NotificationTypes.Warning
+    ) {
+      bg = "ui.gray.x-light-cool";
     }
     return {
       bg,
       display: "flex",
       fontSize: "-1", // slightly smaller than the default size
-      padding: "var(--nypl-space-s) var(--nypl-space-l)",
+      padding: "var(--nypl-space-s)",
```

Comment:
This padding attribute as well as the conditional attributes on lines 18 and 19 should be moved into the styles for the `container`.  The content area here need to act the same as other full width component (ex. `Breadcrums` and the content area of `Template`) where there is a `1280px` max-width and a constant `16px` padding on both sides.  I did some testingm locally and it seems to work fine when the attributes are moved in with the other styles for `container`.

Reply:
I was also curious why this component is using CSS vars and not the theme object.

Reply:
The above makes sense. I think it's how we initially wrote this before using the theme object.